Tribler/community/tunnel/conversion.py

```diff
@@ -58,13 +85,13 @@ def _decode_cell(self, placeholder, offset, data):
 
     def _encode_create(self, message):
         payload = message.payload
-        packet = pack("!IHH20s", payload.circuit_id, len(payload.node_public_key),
-                      len(payload.key), payload.node_id) + payload.node_public_key + payload.key
+        packet = pack("!IHH20s?", payload.circuit_id, len(payload.node_public_key),
+                      len(payload.key), payload.node_id, payload.exit_candidates) + payload.node_public_key + payload.key
```

Comment:
The initiator of a circuit is responsible for choosing the exit-node. It will choose a peer which has the exit-flag enabled. Do we really need this flag here? I'm not so sure.